test/e2e/manifest/testIPAddressAllocation/ipaddressallocation_external.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: crd.nsx.vmware.com/v1alpha1
+kind: IPAddressAllocation
+metadata:
+  name: guestcluster-workers-b
+spec:
+  ipAddressBlockVisibility: External
+  allocationSize: 32
```

Comment:
Sorry one more question, can we set `allocationSize` as 1? Does NSX allow to allocate 1 address with /32 prefix

Reply:
It only allows power of 2 size.